src/gui/QML/settingpages/GlobalSettings.qml

```diff
@@ -241,15 +241,15 @@ Item {
 
                         Slider {
                             id: manualGain
-                            from: 100
-                            to: 0
+                            from: 0
+                            to: 100
                             value: manualGainState
                             stepSize: 1
 
                             Layout.preferredWidth: parent.width
                             onValueChanged: {
                                 if(enableAGC.checked == false)
-                                                cppGUI.inputGainChanged(valueGain)
+                                    cppGUI.inputGainChanged(manualGainState)
```

Comment:
Are you sure that `manualGainState` is correct? I assume it should be `value`?

At the moment I'm redesigning the GUI so there are a lot of untested stuff.

Reply:
Yes, the signal name for the value of the slider sounds odd, but I did not choose that name.
This was maybe you(?) when reworking it the GUI?

I just fixed that forwarding statement of the onValueChanged event and use the actual value name of the slider to make the forwarding work again. And I tried it out before and after my change with my rad1o on a clean install/compile (I even removed the preference plist of welle.io before each test).

After your commit of the file there was this error and gain slider were not working at all:
"qrc:/src/gui/QML/settingpages/GlobalSettings.qml:252: ReferenceError: valueGain is not defined"

After the change the slider works. If you not like the signal name you choose anymore you could change it.